react-native-gutenberg-bridge/ios/Gutenberg.swift

```diff
@@ -51,6 +51,10 @@ public class Gutenberg: NSObject {
     public func toggleHTMLMode() {
         bridgeModule.sendEvent(withName: EventName.toggleHTMLMode, body: nil)
     }
+    
+    public func updateHtml(_ html: String) {
+        bridgeModule.sendEvent(withName: EventName.updateHtml, body: [ "html": html ])
+    }
 }
 
 extension Gutenberg: RCTBridgeDelegate {
```

Comment:
I know it's not a part of this PR, but file name `Gutenberg.swift` doesn't tell anything about the purpose of this class, maybe we can consider re-naming in the future?

Reply:
That seemed a little odd to me too at the beginning but then I started to think that it is a fine name. Gutenberg class provides an interface for all the capabilities of the ReactNative Gutenberg bridge, if you look at that way it is fine to call it Gutenberg because it is THE actual Gutenberg for the parent app. What do you think @etoledom ?

Reply:
FWIW, on the Android side of things, I've introduced the `WPAndroidGlueCode` class that manages the ReactRootView, the RN dependencies and the bridge instance. The name is still poor IMHO and I'll revise it at some point but, it was a deliberate choice to clear from a too ambiguous "Gutenberg". I most probably introduce something like "WPGutenbergGlue" in the near future.